models/proof.rb

```diff
@@ -0,0 +1,34 @@
+class Proof < Sequel::Model
+  def validate
+    super
+
+    errors.add(:mass_g, mass_outweighed_error_msg) if mass_outweighs_offset?
+  end
+
+  def after_create
```

Comment:
Callbacks tend to obscure the underlying logic that gets executed when part of an object's lifecycle gets triggered. For example, it's not clear that these steps happen when `.save` is called in the routes file, nor in what order. Therefore, scaling callbacks can get pretty messy.
What are some other patterns that could be used instead to prevent these issues? What have you seen work best in the past?